doc/faq.rst

```diff
@@ -65,27 +65,35 @@ optimized file format such as HDF5 to reduce data load times. Various libraries
 such as H5Py, PyTables and pandas provides a Python interface for reading and
 writing data in that format.
 
-Can I add this new algorithm that I (or someone else) just published?
--------------------------------------------------------------------------
-No. As a rule we only add well-established algorithms. A rule of thumb is at least
-3 years since publications, 200+ citations and wide use and usefullness. A
-technique that provides a clear-cut improvement (e.g. an enhanced  data
-structure or efficient approximation) on a widely-used method will also be
-considered for inclusion.
-Your implementation doesn't need to be in scikit-learn to be used together
-with scikit-learn tools, though. Implement your favorite algorithm
-in a scikit-learn compatible way, upload it to github and we will list
-it under :ref:`related_projects`.
-Also see selectiveness_.
+What are the inclusion criteria for new algorithms ?
+----------------------------------------------------
+In general, only well-established algorithms are added. A rule of thumb is at
```

Comment:
@jnothman / @amueller Could we skip the `In general, `? That would appear a little impolite, but would emphasize that we are strict with these requirements?